### PR TITLE
MAT-7553: Update render for new displayNames

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7553](https://jira.cms.gov/browse/MAT-7553)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/xsl/qdm_v5_6_measure_details.xsl [hunk 1]
```diff
@@ -335,10 +335,12 @@
                             <xsl:when test="$nameVar = 'Appropriate Use Process'">APPROPRIATE</xsl:when>
                             <xsl:when test="$nameVar = 'Resource Use'">RESOURCE</xsl:when>
                             <xsl:when test="$nameVar = 'Efficiency'">EFFICIENCY</xsl:when>
+                            <xsl:when test="$nameVar = 'Intermediate Outcome'">INTERM-OM</xsl:when>
                             <xsl:when test="$nameVar = 'Intermediate Clinical Outcome'">INTERM-OM</xsl:when>
```
Comment (adongare) [inline src/main/resources/xsl/qdm_v5_6_measure_details.xsl]: we can remove old ones

File: src/main/resources/xsl/qdm_v5_6_measure_details.xsl [hunk 2]
```diff
@@ -335,10 +335,12 @@
...
                             <xsl:when test="$nameVar = 'Outcome'">OUTCOME</xsl:when>
                             <xsl:when test="$nameVar = 'Experience'">EXPERIENCE</xsl:when>
                             <xsl:when test="$nameVar = 'Patient Reported Outcome Performance'">PRO-PM</xsl:when>
```
Comment (adongare) [inline src/main/resources/xsl/qdm_v5_6_measure_details.xsl]: this as well

**Review by adongare — APPROVED** [review]
